src/App.js

```diff
@@ -22,7 +22,7 @@ function App() {
     fetch(url)
       .then((res) => { return res.json() })
       .then((data) => {
-
+        console.log(data)
```

Comment:
Remember to delete this before submitting your project

Reply:
Thanks Jen for spotting this ! Will remove this console.log.